.circleci/integration-tests/master_dag.py

```diff
@@ -68,30 +68,48 @@ def get_report(dag_run_ids: List[str], **context: Any) -> None:
         else:
             airflow_version_message = f"The below run is on Airflow version `{airflow_version} with {airflow_executor} executor`\n\n"
 
-        message_list.append(airflow_version_message)
+        master_dag_deployment_link = f"{os.environ['AIRFLOW__WEBSERVER__BASE_URL']}/dags/example_master_dag/grid?search=example_master_dag"
+        deployment_message = f"\n \n \n <{master_dag_deployment_link}|Link> to the master DAG for the above run on Astro Cloud deployment: Master DAG \n"
```

Comment:
```suggestion
        deployment_message = f"\n \n \n <{master_dag_deployment_link}|Link> to the master DAG for the above run on Astro Cloud deployment \n"
```

Since the link is in the beginning and we mentioned already master DAG in the message, no need for additional text